1.9/installing/custom/configuration/configuration-parameters.md

```diff
@@ -102,25 +102,21 @@ For more information, see the [security documentation](https://docs.mesosphere.c
 The dictionary of Docker credentials to pass. 
 
 - If unset, a default empty credentials file is created at `/etc/mesosphere/docker_credentials` during DC/OS install. A sysadmin can change credentials as needed. A `systemctl restart dcos-mesos-slave` or `systemctl restart dcos-mesos-slave-public` is required for changes to take effect.
```

Comment:
Ditto.

Reply:
@chhsia0 Please see amended note and separate cluster_docker_credentials_enabled description.